agile.sh

```diff
@@ -7,7 +7,7 @@ requires:
   - boost
   - lhapdf5
   - HepMC
-  - Python
+  - Python-modules
```

Comment:
this reminds me about the issue that the compilation of `Python-modules` does not work on mac, so there it's a system requirement. Could that be reflected in the defaults? #704, #724

Reply:
You have a point. This _is_ a system requirement everywhere but on slc5 and slc6 and should be treated as such.